test/serializers/meta_test.rb

```diff
@@ -125,6 +127,20 @@ def test_meta_is_present_on_arrays_with_root
         }
         assert_equal(expected, actual)
       end
+
+      def test_meta_is_set_with_direct_attributes
+        MetaBlogSerializer.meta stuff: 'value'
+        blog_meta_serializer = MetaBlogSerializer.new(@blog)
+        assert_equal(blog_meta_serializer.meta, stuff: 'value')
+      end
+
+      def test_meta_is_set_with_block
+        MetaBlogSerializer.meta do
+          { articles_count: object.articles.count }
+        end
+        blog_meta_serializer = MetaBlogSerializer.new(@blog)
+        assert_equal(blog_meta_serializer.meta, articles_count: @blog.articles.count)
+      end
```

Comment:
I'd like to validate that meta creates valid JSON API objects.  Do you think this PR is a good spot for that?